homepairs/homepairsUI/src/modals/property-modals/EditPropertyModal/EditPropertyModalBase.tsx

```diff
@@ -88,7 +88,7 @@ function setInputStyles(colorTheme?: BaseStyles.ColorTheme){
             justifyContent: 'center',
             alignItems: 'center',
             alignSelf: 'center',
-            flex: 1,
+            // flex: 1,
```

Comment:
why comment / not just erase?

Reply:
Certain many of these issues with alignment had worked for certain platforms but had been messing with things on others. I was playing around with many flexes. I could simply remove them now since things are looking nicely on all platforms now.